playbooks/ceph.yml

```diff
@@ -63,7 +63,7 @@
   become: true
   vars:
     cifmw_admin_user: ceph-admin
-    ansible_ssh_private_key_file: "{{ lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY') }}"
+    ansible_ssh_private_key_file: "{{ hostvars[(groups[cifmw_ceph_target | default('computes')] | default([]) | first)]['ansible_ssh_private_key_file'] | default(lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY')) }}"
```

Comment:
care to split a bit that line? It's barely readable...
Something like:
```yaml
vars:
  _target_group: "{{ cifmw_ceph_target | default('computes') }}"
  _target: "{{ groups[_target] | default([]) | first }}"
 ansible_ssh_private_key_file: >-
  {{
    hostvars[_target]['ansible_ssh_private_key_file'] |
    default(lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY'))
  }}
```

Also, I'm not sure the `| default([]) | first` works, iirc `first` will crash because there's no first element in an empty list... Care to validate that on the side?

Reply:
done